app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -9,10 +11,10 @@
 //
 //## Gray and brand colors for use across Bootstrap.
 
-// $gray-base:              #000
+$gray-base:              #000;
 // $gray-darker:            lighten($gray-base, 13.5%) // #222
 // $gray-dark:              lighten($gray-base, 20%)   // #333
-// $gray:                   lighten($gray-base, 33.5%) // #555
+$gray:                   lighten($gray-base, 33.5%); // #555
```

Comment:
@Flaburgan I'ts not `$grey` [it's a new color](https://github.com/diaspora/diaspora/pull/6309/files#diff-b48a3c07783ee2256d72153189c9743bR28)